lib/util/util_spawn.c

```diff
@@ -20,13 +20,13 @@ extern char **environ;
 #define STDERR_FILENO 2
 
 
-static void __init_attributes(posix_spawnattr_t * attributes) {
+static void spawn_init_attributes__(posix_spawnattr_t * attributes) {
```

Comment:
I would guess that if you absolutely need to annotate the function names, then _one_ trailing underscore would suffice?